Clients/GUI/Scripts/BotManager.cs

```diff
@@ -0,0 +1,57 @@
+﻿using DTO;
+using DTO.RestRequests;
+using Kernel;
+using Kernel.Enums;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace GUI.Scripts
+{
+    public class BotManager
+    {
+        public static async Task CreateBot(UserToken userToken, CreateBotRequest request)
+        {
+            const string url = "https://localhost:5009/operations/bot/create";
+
+            var client = new RestClient<CreateBotRequest, bool>(url, RestRequestType.POST, userToken);
+
+            await client.ExecuteAsync(request);
+        }
+
+        public static async Task DeleteBot(UserToken userToken, DeleteBotRequest request)
+        {
+            const string url = "https://localhost:5009/operations/bot/delete";
+
+            var client = new RestClient<DeleteBotRequest, bool>(url, RestRequestType.DELETE, userToken);
+
+            await client.ExecuteAsync(request);
+        }
+
+        public static async Task RunBot(UserToken userToken, RunBotRequest request)
```

Comment:
Maybe run and disable should be the one request with bool field?

Reply:
I don't think so, because it will do code more difficult.